packages/editor/src/components/editor-help/index.native.js

```diff
@@ -113,6 +124,9 @@ function EditorHelpTopics( { close, isVisible, onClose } ) {
 										} }
 									>
 										<PanelBody>
+											<Text style={ sectionTitle }>
+												{ __( 'The basics' ) }
```

Comment:
Should we update the added titles to title case?

Reply:
I think this is a great @kyleaparker question! I've requested a design review, so let's leave this open until she can provide input. Does that sound good @hypest?